nav2_bringup/launch/bringup_launch.py

```diff
@@ -38,6 +40,27 @@ def generate_launch_description():
     use_sim_time = LaunchConfiguration('use_sim_time')
     params_file = LaunchConfiguration('params_file')
     autostart = LaunchConfiguration('autostart')
+    use_composition = LaunchConfiguration('use_composition')
+
+    # Map fully qualified names to relative ones so the node's namespace can be prepended.
+    # In case of the transforms (tf), currently, there doesn't seem to be a better alternative
+    # https://github.com/ros/geometry2/issues/32
+    # https://github.com/ros/robot_state_publisher/pull/30
+    # TODO(orduno) Substitute with `PushNodeRemapping`
+    #              https://github.com/ros2/launch_ros/issues/56
+    remappings = [('/tf', 'tf'),
```

Comment:
I'm not entirely sure why you added this block L43-63, I know other files have it, but was it needed here as well for some reason new to composition?

Reply:
yes, it's used by `composed_bringup` node : https://github.com/ros-planning/navigation2/blob/36641f262c0d649ccffc9305eb0910e6154ed9f5/nav2_bringup/launch/bringup_launch.py#L130-L136